c2corg_ui/static/js/auth/auth.js

```diff
@@ -186,7 +186,11 @@ app.AuthController.prototype.successLogin_ = function(remember, response) {
  */
 app.AuthController.prototype.register = function() {
   var alerts = this.alerts_;
-  this.api_.register(this.scope_['register']).then(function() {
+  var lang = window.localStorage.getItem('interface_lang') || 'fr';
```

Comment:
Thanks for spotting this. Strangely, I had the pair in my localstorage!
This is now fixed.